java/src/Ice/src/main/java/com/zeroc/Ice/ProtocolVersion.java

```diff
@@ -118,33 +120,28 @@ static public ProtocolVersion ice_read(InputStream istr)
         return v;
     }
 
-    static public void ice_write(OutputStream ostr, int tag, java.util.Optional<ProtocolVersion> v)
-    {
-        if(v != null && v.isPresent())
-        {
-            ice_write(ostr, tag, v.get());
-        }
-    }
-
-    static public void ice_write(OutputStream ostr, int tag, ProtocolVersion v)
+    static public void ice_write(OutputStream ostr, int tag, @Nullable ProtocolVersion v)
```

Comment:
The marshaling code for non-generated types (that were previously generated) should be removed, not updated.